source/languages.ttl

```diff
@@ -0,0 +1,3171 @@
+
+<fil> a schema:Language ;
+    skos:notation "fil",
+        "fil"^^kbv:ISO639-2,
+        "fil"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Filipino"@sv .
+
+<fin> a schema:Language ;
+    skos:notation "fin",
+        "fin"^^kbv:ISO639-2,
+        "fin"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Finska"@sv .
+
+<fit> a schema:Language ;
+    owl:sameAs <9mk> ;
+    rdfs:comment "9mk Librisdefinierad MARC-kod"@sv ;
+    skos:notation "fit",
+        "fit"^^kbv:ISO639-3,
+        "9mk"^^kbv:LibrisMarcLanguageCode ;
+    skos:prefLabel "Meänkieli"@sv ;
+    skos:prefLabel "Meänkieli"@en .
+
+<fiu> a schema:Language ;
+    skos:notation "fiu",
+        "fiu"^^kbv:ISO639-2,
+        "fiu"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Finsk-ugriska språk (övriga)"@sv .
+
+<fon> a schema:Language ;
+    skos:notation "fon",
+        "fon"^^kbv:ISO639-2,
+        "fon"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Fon"@sv .
+
+<fre> a schema:Language ;
+    skos:notation "fre",
+        "fre"^^kbv:ISO639-2,
+        "fre"^^kbv:ISO639-2-B,
+        "fra"^^kbv:ISO639-2-T,
+        "fra"^^kbv:ISO639-3 ;
+    skos:prefLabel "Franska"@sv .
+
+<frm> a schema:Language ;
+    rdfs:comment "Ca 1300-1600"@sv ;
+    skos:notation "frm",
+        "frm"^^kbv:ISO639-2,
+        "frm"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Medelfranska"@sv .
+
+<fro> a schema:Language ;
+    rdfs:comment "Ca 842-1300"@sv ;
+    skos:notation "fro",
+        "fro"^^kbv:ISO639-2,
+        "fro"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Fornfranska"@sv .
+
+<frr> a schema:Language ;
+    skos:notation "frr",
+        "frr"^^kbv:ISO639-2,
+        "frr"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Nordfrisiska"@sv .
+
+<frs> a schema:Language ;
+    skos:notation "frs",
+        "frs"^^kbv:ISO639-2,
+        "frs"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Östfrisiska"@sv .
+
+<fry> a schema:Language ;
+    skos:notation "fry",
+        "fry"^^kbv:ISO639-2,
+        "fry"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Frisiska"@sv .
+
+<ful> a schema:Language ;
+    skos:notation "ful",
+        "ful"^^kbv:ISO639-2,
+        "ful"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Fula"@sv .
+
+<fur> a schema:Language ;
+    skos:notation "fur",
+        "fur"^^kbv:ISO639-2,
+        "fur"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Friuliska"@sv .
+
+<gaa> a schema:Language ;
+    skos:notation "gaa",
+        "gaa"^^kbv:ISO639-2,
+        "gaa"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Ga"@sv .
+
+<gay> a schema:Language ;
+    skos:notation "gay",
+        "gay"^^kbv:ISO639-2,
+        "gay"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Gayo"@sv .
+
+<gba> a schema:Language ;
+    skos:notation "gba",
+        "gba"^^kbv:ISO639-2,
+        "gba"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Gbaya"@sv .
+
+<gem> a schema:Language ;
+    skos:notation "gem",
+        "gem"^^kbv:ISO639-2,
+        "gem"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Germanska språk (övriga)"@sv .
+
+<geo> a schema:Language ;
+    skos:notation "geo",
+        "geo"^^kbv:ISO639-2,
```

Comment:
```suggestion
        "geo"^^kbv:ISO639-2,
        "kat"^^kbv:ISO639-2,
```

Reply:
ISO-639-2 = ISO-639-2/B ∪  ISO-639-2/T so both codes should always exist as `^^kbv:ISO639-2`

Reply:
We have all three `kbv:ISO639-2`, `bv:ISO639-2-T`, `bv:ISO639-2-B`

Reply:
Let's drop the redundant `kbv:ISO639-2` if there are specific `kbv:ISO639-2-B` and/or `kbv:ISO639-2-T` codes (those types are already defined as `rdfs:subClassOf :ISO639-2`).

Reply:
Reply

Reply:
Reply